homeassistant/components/aws/notify.py

```diff
@@ -12,17 +12,15 @@
     ATTR_TITLE_DEFAULT,
     BaseNotificationService,
 )
-from homeassistant.const import CONF_NAME, CONF_PLATFORM
-from homeassistant.helpers.json import JSONEncoder
-
-from .const import (
-    CONF_CONTEXT,
-    CONF_CREDENTIAL_NAME,
+from homeassistant.const import (
+    CONF_NAME,
+    CONF_PLATFORM,
     CONF_PROFILE_NAME,
-    CONF_REGION,
     CONF_SERVICE,
-    DATA_SESSIONS,
 )
+from homeassistant.helpers.json import JSONEncoder
+
+from .const import CONF_CONTEXT, CONF_CREDENTIAL_NAME, CONF_REGION, DATA_SESSIONS
```

Comment:
Not sure why these need to be moved.

Reply:
CONF_PROFILE_NAME and CONF_SERVICE exist in homeassistant/const.py. That is why they are moved. The rest is just formatting but can be confusing.